package.json

```diff
@@ -31,7 +38,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/mojaloop/auth-service.git"
+    "url": "git+https://github.com/mojaloop/AuthService.git"
```

Comment:
Is this supposed to be AuthService or find/replace mishap?

Reply:
Good eye you have @kleyow It's find/replace mishap... fixed